src/main/webapp/WEB-INF/pages/manage/home.ftl

```diff
@@ -6,14 +6,10 @@
 <#include "/WEB-INF/pages/macros/forms.ftl"/>
 <#include "/WEB-INF/pages/macros/popover.ftl">
 <#include "/WEB-INF/pages/macros/resourcesTable.ftl"/>
-<script type="text/javascript" src="${baseURL}/js/jquery/jquery-3.5.1.min.js"></script>
```

Comment:
This line was removed to the library jquery.js, and it is a different version. But, looks like the right change.

Reply:
I dont know to this is affecting. We have to take in mind this.